src/pynwb/__init__.py

```diff
@@ -4,28 +4,24 @@
 import os.path
 from copy import deepcopy
 from warnings import warn
-
 import h5py
+from pkg_resources import resource_filename
```

Comment:
This was probably intentionally a function-level import because you don't want to pull it into the pynwb module

Reply:
Ohhh I see.

Reply:
This code is always run though (see line 102), so wouldn't it be pulled into the pynwb module anyway?

Reply:
Never mind. I tested it and you are right. It is not pulled in if it is in the function. Will change.